nlfyapp/src/features/component/sermon.component.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable prettier/prettier */
+import React from "react";
+import { HeaderSermon } from "./sermonHeader.component";
+import { SermonVideo } from "./sermonVideo.component";
+
+export const Sermon = () => {
+    return (
+        <>
+            <HeaderSermon />
```

Comment:
Can you try to create a common Header component for Home Screen and re use for each section ?

Reply:
I have made the changes please review :
1) created a common Header Component -> homeScreenHeader.component.js
2) created a new folder homeScreenComponent in features/component and pused all the home screen related component into it.